psychopy/data.py

```diff
@@ -3464,8 +3464,8 @@ class QuestHandler(StairHandler):
         # create staircase object
         # trying to find out the point where subject's response is 50 / 50
         # if wanted to do a 2AFC then the defaults for pThreshold and gamma
-        # are good
-        staircase = data.QuestHandler(staircase._nextIntensity, 0.2,
+        # are good. As start value, we'll use 50% contrast, with SD = 20%
+        staircase = data.QuestHandler(0.5, 0.2,
             pThreshold=0.63, gamma=0.01,
```

Comment:
It [does say](https://github.com/psychopy/psychopy/blob/c564dbd61be91bed6ab47c50f9e9ae8547c2883b/psychopy/data.py#L3511):
```
        Typical values for pThreshold are:
            * 0.82 which is equivalent to a 3 up 1 down standard staircase
            * 0.63 which is equivalent to a 1 up 1 down standard staircase
                (and might want gamma=0.01)
```
But in the example code, we talk about 2-AFC, so I believe we should also provide a "real-world proof" 2-AFC example, which would typically target the 75% threshold... Just my personal view!